client/my-sites/stats/sections/all-time-views-section/style.scss

```
@@ -1,6 +1,27 @@
 @import "calypso/my-sites/stats/modernized-stats-table-styles";
 
+.stats__all-time-views-section {
+	.highlight-cards-list {
+		display: block;
+	}
+
+	.highlight-card-heading {
+		flex-wrap: wrap;
+		gap: 16px
```

```suggestion
		gap: 8px 16px;
```
Maybe a smaller `row-gap`?